### ISSUE TITLE
Feat: [Question]  model class 구성 코드에 관하여

### ISSUE THREAD

**Issue by unanchoi** [CUAI-CAU/PyTorch-Study-T2_2022#1]
강의를 보면서,

model class를 구성하는 2개의 메소드가 __init__과 forward 메소드가 반드시 필요하더라고요!!
강의 예제는 간단한 데이터로 구성되어있지만, 혹시 추가로 더 필요한 메소드가 있을지 궁금합니다!

또한 Dataloader와 같은 다른 클래스에서도 이런 필수적인 메소드가 존재하는지 궁금합니다.

**Comment by linkyouhj** [comment]
Custom Dataset은 __init__, __len__, __getitem__ 세가지 함수 구현이 반드시 필요합니다. DataLoader는 torch.utils.data에서 import 후 다음과 같이 선언합니다.

train_dataloader = DataLoader(training_data, batch_size=64, shuffle=True)

이런 식으로 선언해서 model이나 dataset과는 형태가 다소 다르네요.
